src/mortality.jl

```diff
@@ -0,0 +1,195 @@
+const HILLCOEFFICIENT = Param(0.1; bounds=(0.0, 100))
+const MEDIAN_LOGLOGISTIC = Param(0.1; bounds=(0.0, 100))
+const THRESHOLD_EXPOSURE = Param(0.0; bounds=(0.0, 1e9))
+
+"""
+Extends CellRule for rules of survival effect
+For best performance these should be chained with other
+CellRule or following an NeighborhoodRule.
+"""
+abstract type Mortality{R,W} <: CellRule{R,W} end
+
+"""
+    LoglogisticMortality(; median, hillcoefficient, timestep)
+    LoglogisticMortality{R}(; median, hillcoefficient, timestep)
+    LoglogisticMortality{R,W}(; median, hillcoefficient, timestep)
+
+Loglogistic mortality based on median,  ``α`` and hill coefficient ``β``.
+
+Cumulative function of loglogistic: 
+```math
+F(x; α, β) =  x^β/(α^β + x^β)
+```
+where α>0 is the scale and β>0 is the shape.
```

Comment:
shoudld these be ``α>0`` and ``β>0`` with `` ?

Maybe check that all this math prints properly by building the docs locallly. `julia make.jl` inside `docs` folder, then open the `index.html` file in build.

Reply:
Ok, yes, I was wondering if math looked properly.

Reply:
yeah it's annoying not being able to check the docs on a PR on github